### PR TITLE
310 sources should work without prefixdir

### PR DESCRIPTION
**Pull request by DavieReid**

### CODE REVIEW

File: packages/plugins/src/LazyPagePlugin.ts [hunk 1]
```diff
@@ -66,6 +66,8 @@ const LazyPagePlugin: PluginType<LazyPagePluginPage, LazyPagePluginOptions> = {
       cwd: '/',
       onlyFiles: true
     });
+
+    console.log('lazy beforeSend');
```
Comment (DavieReid) [inline packages/plugins/src/LazyPagePlugin.ts:70]: will remove

Comment (DavieReid) [reply packages/plugins/src/LazyPagePlugin.ts]: DONE

File: packages/site/mosaic.config.js [hunk 2]
```diff
@@ -38,12 +38,12 @@ module.exports = deepmerge(siteConfig, {
      * Access from your browser as http://localhost:3000/mosaic
      */
     {
-      disabled: process.env.NODE_ENV !== 'development',
+      // disabled: process.env.NODE_ENV !== 'development',
```
Comment (mark-tate) [inline packages/site/mosaic.config.js:41]: commented out code

File: packages/site/mosaic.config.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const siteConfig = {
     ...mosaicConfig.plugins,
     {
       modulePath: '@jpmorganchase/mosaic-plugins/SidebarPlugin',
-      options: { rootDirGlob: '*/*' }
+      options: { rootDirGlob: '*' }
```
Comment (mark-tate) [inline packages/site/mosaic.config.js:10]: this flips us back to a single sidebar for everything..
I think we were going to converge with Salt.. is this a change we still want ?

File: packages/plugins/src/BreadcrumbsPlugin.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ const BreadcrumbsPlugin: PluginType<BreadcrumbsPluginPage, BreadcrumbsPluginOpti
         let currentPage = page;
         let parentDir = path.posix.normalize(path.posix.dirname(currentPage.fullPath));
 
-        while (currentPage !== undefined) {
+        while (currentPage !== undefined && currentPage.fullPath !== '/index.mdx') {
```
Comment (mark-tate) [inline packages/plugins/src/BreadcrumbsPlugin.ts:27]: I remember JP be vocal about not coupling us to specific file types ?
Some of our configs have things like 'extensions' in them, so hardcoding a filetype in source, feels very specific ?
Maybe our position has changed on this but what's the rule going forward and how does this affect config values ?

File: packages/site/mosaic.config.js (hunk 3 above)
Comment (DavieReid) [reply packages/site/mosaic.config.js:10]: There is no sidebar without this...

File: packages/plugins/src/BreadcrumbsPlugin.ts (hunk 4 above)
Comment (DavieReid) [reply packages/plugins/src/BreadcrumbsPlugin.ts:27]: Will rework

Comment (DavieReid) [reply packages/plugins/src/BreadcrumbsPlugin.ts]: Fixed

File: packages/site/mosaic.config.js (hunk 2 above)
Comment (DavieReid) [reply packages/site/mosaic.config.js]: removed this code...it was only for testing

File: packages/site/next.config.js [hunk 5]
```diff
@@ -50,17 +50,7 @@ module.exports = {
...
-      },
-      {
-        source: '/mosaic',
-        destination: '/mosaic/index',
-        permanent: true
-      },
-      {
-        source: '/local',
-        destination: '/local/index',
+        destination: '/index',
```
[the remark this answers is not in the document]
Comment (DavieReid) [reply packages/site/next.config.js]: removed this code...it was only for testing

File: packages/site/mosaic.config.js (hunk 3 above)
Comment (DavieReid) [reply packages/site/mosaic.config.js]: removed this code...it was only for testing

**Review by lilyvc — APPROVED** [review]

**Pull request merged**